<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configurationDialog</class>
 <widget class="QDialog" name="configurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelBaitImagePath">
       <property name="text">
        <string>Path to &amp;bait image files</string>
       </property>
       <property name="buddy">
        <cstring>editBaitImagePath</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFishImagePath">
       <property name="text">
        <string>Path to &amp;fish image files</string>
       </property>
       <property name="buddy">
        <cstring>editFishImagePath</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editFishImagePath"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editBaitImagePath">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Base pathname to bait image files on local hard drive&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editCatchImagePath"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCatchImagePath">
       <property name="text">
        <string>Path to &amp;catch image files</string>
       </property>
       <property name="buddy">
        <cstring>editCatchImagePath</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editLocationImagePath">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelLocationImagePath">
       <property name="text">
        <string>Path to &amp;location image files</string>
       </property>
       <property name="buddy">
        <cstring>editLocationImagePath</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialogButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editBaitImagePath</tabstop>
  <tabstop>editFishImagePath</tabstop>
  <tabstop>editCatchImagePath</tabstop>
  <tabstop>editLocationImagePath</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>configurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>configurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
